package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "judge-d",
+    "version": "0.0.1",
+    "description": "CLI for publishing and validating contract tests using judge-d API",
+    "keywords": [
+        "pacts",
+        "judge-d",
+        "contracts"
+    ],
+    "main": "./lib/index.js",
+    "bin": {
+        "judge-d": "judge-d.js"
+    },
+    "files": [
+        "bin",
+        "lib"
+    ],
+    "scripts": {
+        "build": "tsc -p tsconfig.json",
+        "test": "jest",
+        "release": "semantic-release",
+        "prepare": "husky install"
```

Comment:
Why is this needed?

Reply:
It was also autogenerated by husky - it edits config inside .git directory and changes default hooksPath to ".husky". Without it husky hooks won't run.